k9mail/src/main/java/com/fsck/k9/planck/ui/fragments/AccountSetupBasicsFragmentBase.kt

```diff
@@ -222,5 +282,7 @@ abstract class AccountSetupBasicsFragmentBase : Fragment() {
             "com.fsck.k9.AccountSetupBasics.checkedIncoming"
         private const val REQUEST_CODE_CHECK_SETTINGS =
             AccountSetupCheckSettings.ACTIVITY_REQUEST_CODE
+        private const val REQUEST_CODE_OAUTH = Activity.RESULT_FIRST_USER + 1
```

Comment:
Is Activity.RESULT_FIRST_USER a constant? If so it is a bit strange to init like this a constant with +1 as it can be error prone if you change constant values (I mean it defeats a bit the purpose of having a name you can change in one place with the math operation)

Reply:
Updated.